core/store/lib/search/adapter/graphql/queries/cmsHierarchy.gql

```diff
@@ -0,0 +1,14 @@
+query cmsHierarchies ($filter: CmsInput) {
```

Comment:
should be in module/queries

Reply:
This gql is related to the certain searchAdapter. Not sure if it should be in the module/queries

Reply:
Same as above - it should be within the module only. We need to have a capability of extending graphQL entities for custom ones from the module level